Update Debian based dependencies in README (#460)
projecthamster/hamster: README.md

###### Ubuntu (tested in 19.04 and 18.04)

> ```bash

Reviewer: ```suggestion
sudo apt install gettext intltool gconf2 gir1.2-gconf-2.0 python3-gi-cairo python3-distutils python3-dbus python3-xdg
```

sudo apt install gconf2 gir1.2-gconf-2.0 python3-gi-cairo python3-distutils python3-dbus python3-xdg

> sudo apt install gettext intltool gnome-doc-utils yelp

Reviewer: ```suggestion
# and for documentation:
sudo apt install gnome-doc-utils yelp
```
Added a comment to make the lines separation purpose clearer
